# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/printing/nsPrintJob.h
# Commit: 5cd20e839a73
# Full Hash: 5cd20e839a7322ad075dcc6d88585a28d36db87d
# Author: Jonathan Watt <jwatt@jwatt.org>
# Date: 2022-05-23 09:52:44
# Regressor Bug: 1770539
# File Overlap Count: 1
# Description:
#   Bug 1770539 p2 - Move nsPrintData::mShrinkToFit to nsPrintJob. r=emilio
#   
#   Depends on D146973
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D146974
# ==============================================================================

diff -r f6e9ace877bd -r 5cd20e839a73 layout/printing/nsPrintJob.h
--- a/layout/printing/nsPrintJob.h	Sun May 22 09:37:55 2022 +0000
+++ b/layout/printing/nsPrintJob.h	Sun May 22 09:37:56 2022 +0000
@@ -266,6 +266,7 @@
   bool mIsCreatingPrintPreview = false;
   bool mIsDoingPrinting = false;
   bool mDidLoadDataForPrinting = false;
+  bool mShrinkToFit = false;
   bool mDoingInitialReflow = false;
   bool mIsDestroying = false;
   bool mDisallowSelectionPrint = false;
